plugins/bean-validation/src/test/resources/bean-validation-test.xml

```diff
@@ -20,8 +20,8 @@
  */
 -->
 <!DOCTYPE xwork PUBLIC
-        "-//Apache Struts//XWork 2.0//EN"
-        "http://struts.apache.org/dtds/xwork-2.0.dtd"
+        "-//Apache Struts//XWork 2.5//EN"
+        "http://struts.apache.org/dtds/xwork-2.5.dtd"
```

Comment:
👍

Reply:
...<allowed-methods>... was the reason :-)